content/content-strategy/audit-content/prove-value/section4.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: text/textblock
+---
+Your manager or stakeholder may be unaware of how your websites measure against:
+  * [content best practice](https://guides.service.gov.au/content-guide/), for example, user-centred, accessible content written in plain English and 
+  * government requirements, for example, the [Digital Service Standard](https://guides.service.gov.au/digital-service-standard/).
```

Comment:
Because this is written as an absolute path, it may feature the external link icon at the end. Given it's part of the site, should it actually be a _relative_ link instead? @Phend-117

Reply:
Correct. Will work, but should be relative.

Reply:
Trisha fixed:  /content-guide/ will work when pushed to staging. DSS is ok as relative.